app/assets/javascripts/bastion/components/nutupane.factory.js

```diff
@@ -61,7 +62,9 @@ angular.module('Bastion.components').factory('Nutupane',
             params.paged = true;
             params.page = $location.search().page || 1;
             params['per_page'] = $location.search()['per_page'] || entriesPerPage;
-
+            //Additional param to override initial load
+            nutupaneParams = nutupaneParams || {};
+            self.overrideAutoLoad = nutupaneParams.overrideAutoLoad || false;
```

Comment:
> Bit of a minor point, but the way this is used I think disabled might be better than override here. Only because override implies that I am able to specify my own behavior whereas this is more of a flag I can set to disable autoLoading.

Renamed the flag to disableAutoLoad and added an additional test. Please review.